tests/test_acasclient.py

```diff
@@ -2352,6 +2353,191 @@ def test_50_delete_ls_thing(self):
+        updated_name = "Benzoic Acid"
+        updated_abbrev = "BzCOOH"
+        dryrun = "true"
+        edit_response = self.client.edit_salt(id, dryrun, updated_abbrev, updated_name, mol_structure)
+        self.assertIn("Duplicate salt formula. Another salt exists with the same formula.", edit_response[4]["message"])
+        # Remove The Salts Registered For This Test 
+        delete_response = self.client.delete_salt(id)
+        self.assertIn("There were no lot dependencies found for this salt.", delete_response)
+        search_result = self.client.search_salts("Iodine")
+        id = search_result[0]["id"]
+        delete_response = self.client.delete_salt(id)
+        self.assertIn("There were no lot dependencies found for this salt.", delete_response)
+
+    def test_64_edit_salt_check_update_lot_molecular_weight(self):
+        self.registerTestSalt() 
+        search_result = self.client.search_salts("Benzoic Acid")
+        original_weight = search_result[0]["molWeight"]
+        id = search_result[0]["id"]
+        updated_struct = "\n  Ketcher 08172213572D 1   1.00000     0.00000     0\n\n 11 11  0     1  0            999 V2000\n    5.2250   -4.6500    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0\n    6.0910   -5.1500    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0\n    6.0910   -6.1500    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0\n    5.2250   -6.6500    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0\n    4.3590   -6.1500    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0\n    4.3590   -5.1500    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0\n    6.9570   -4.6500    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0\n    7.8231   -5.1500    0.0000 O   0  0  0  0  0  0  0  0  0  0  0  0\n    6.9570   -3.6500    0.0000 O   0  0  0  0  0  0  0  0  0  0  0  0\n    7.8231   -6.1500    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0\n    8.6891   -6.6500    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0\n  1  2  1  0     0  0\n  2  3  2  0     0  0\n  3  4  1  0     0  0\n  4  5  2  0     0  0\n  5  6  1  0     0  0\n  6  1  2  0     0  0\n  2  7  1  0     0  0\n  7  8  1  0     0  0\n  7  9  2  0     0  0\n  8 10  1  0     0  0\n 10 11  1  0     0  0\nM  END\n"
+        updated_name = "Ethyl benzoate"
+        updated_abbrev = "BzCOOCH2CH3"
+        dryrun = "false"
+        edit_response = self.client.edit_salt(id, dryrun, updated_abbrev, updated_name, updated_struct)
```

Comment:
I'm wondering if you need this line if edit_response isn't checked like search_result/delete_response? I'm not too familiar with this so more curious than anything.